transports/webrtc/src/tokio/udp_mux.rs

```diff
@@ -398,8 +398,11 @@ impl UDPMuxNewAddr {
 
                             continue;
                         }
-                        Poll::Ready(Err(err)) if err.kind() == ErrorKind::TimedOut => {}
                         Poll::Pending => {}
+                        Poll::Ready(Err(err)) if err.kind() == ErrorKind::TimedOut => {}
+                        Poll::Ready(Err(err)) if err.kind() == ErrorKind::ConnectionReset => {
+                            log::debug!("ConnectionReset by remote client {err:?}")
```

Comment:
```suggestion
                            log::debug!("ConnectionReset on socket: {err:?}")
```

I am not sure we can actually assume that the remote itself reset the connection. It might as well be in the networking stack of our machine or any router in between. Please correct me if I am wrong.